awslogs/bin.py

```diff
@@ -55,6 +55,12 @@ def add_common_arguments(parser):
                             default=os.environ.get('AWS_REGION', None),
                             help="aws region")
 
+        parser.add_argument("--aws-endpoint-url",
```

Comment:
Making this argument read the environment like the other arguments would be ideal